src/Map.jsx

```diff
@@ -257,9 +264,11 @@ class Map extends Component {
           )
           : null,
         getRoutesLayer(geojson),
+        !(Object.keys(subroute).length === 0) && subroute.constructor === Object
+          ? getSubRoutesLayer(subroute)
```

Comment:
@EddyIonescu 
Is this logic flawed? Is my PathLayer not getting added to the list of layers to render?